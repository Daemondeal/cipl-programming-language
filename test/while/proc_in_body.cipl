while true: proc foo(): pass