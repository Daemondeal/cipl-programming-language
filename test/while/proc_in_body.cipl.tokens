Token          Lexeme         Line 
WHILE          while          1    
TRUE           true           1    
COLON          :              1    
PROC           proc           1    
IDENTIFIER     foo            1    
LEFT_PAREN     (              1    
RIGHT_PAREN    )              1    
COLON          :              1    
PASS           pass           1    
LINE_END       \n             1    
EOF                           1    
