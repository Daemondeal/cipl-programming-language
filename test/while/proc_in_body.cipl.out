::stdout::

::stderr::
Error on line 1 at `proc`: Unexpected token.
